docs/plugin_developers_guide/10_testing.rst

```diff
@@ -413,42 +413,31 @@ You can also use a similar approach to that of running gdb to use the *strace* c
 Memory Leaks and Corruptions
 ----------------------------
 
-The same approach can be used to make use of the *valgrind* command to find memory corruption and leak issues in your plugin
+Fledge has integrated supported that allows south and north services to be run using the *valgrind* tool.  This tool makes it easy to find memory corruption and leak issues in your plugin
 
   - Create the service that uses your plugin, say a south service and name that service as you normally would.
    
-  - Disable that service from being started by Fledge
+  - Shutdown Fledge
 
-  - Use the fledge status script to find the arguments to pass the service
+  - If using a south service to test your plugin set the environment variable VALGRIND_SOUTH to be the name of the service you just defined.
 
-    .. code-block:: console
+  - Start Fledge using the fledge script in the scripts directory.
```

Comment:
the fledge script in the scripts directory -> the _fledge_ script in the _scripts_ directory

The convention on the page seems to be that commands and directories are in italics.